eZ/Bundle/PlatformInstallerBundle/src/Command/InstallPlatformCommand.php

```diff
@@ -127,7 +136,10 @@ private function checkCreateDatabase(OutputInterface $output)
         );
         try {
             $bufferedOutput = new BufferedOutput();
-            $this->executeCommand($bufferedOutput, 'doctrine:database:create --if-not-exists');
+            $repositoryConfig = $this->repositoryConfigurationProvider->getRepositoryConfig();
+            $connectionName = $repositoryConfig[self::REPOSITORY_STORAGE][self::REPOSITORY_CONNECTION];
```

Comment:
Could you please move this logic to the dedicated private method e.g. `getCurrentConnectionName`?

Reply:
@adamwojs done!